android/sdl_android/src/main/java/com/smartdevicelink/encoder/VirtualDisplayEncoder.java

```diff
@@ -203,9 +204,14 @@ public void run() {
                     inputSurface = prepareVideoEncoder();
 
                     // Create a virtual display that will output to our encoder.
-                    virtualDisplay = mDisplayManager.createVirtualDisplay(TAG,
-                            streamingParams.getResolution().getResolutionWidth(), streamingParams.getResolution().getResolutionHeight(),
-                            streamingParams.getDisplayDensity(), inputSurface, DisplayManager.VIRTUAL_DISPLAY_FLAG_PRESENTATION);
+                    if (Build.VERSION.SDK_INT >= VERSION_CODES.LOLLIPOP && virtualDisplay != null) {
```

Comment:
@kostyaBoss Now that we have reverted the change to the shutdown method of VirtualDisplayEncoder do we still need this change here?
If not I believe all changes in this file can be reverted